nix/haskell-pins.nix

```diff
@@ -172,6 +172,13 @@ let
         sha256 = "1w23yz2iiayniymk7k4g8gww7268187cayw0c8m3bz2hbnvbyfbc";
       };
     };
+    swagger2 = {
```

Comment:
Why do we need exactly this version of `swagger2`?

Reply:
@mdimjasevic asked the same thing. The swagger specification supports [extensions](https://swagger.io/docs/specification/2-0/swagger-extensions/), but `swagger2` doesn't. Julia and I decided to patch swagger2 with the functionality and have [upstreamed it](https://github.com/GetShopTV/swagger2/pull/245), but it's unclear if the library is maintained.